codice/ddf
# DDF-4170 Moving search form management out of workspace

#### What does this PR do?
Moves Search form management outside the workspace. As well as removing search forms as an experimental feature.  
#### Who is reviewing it? 
@GabrielFabian @willwill96 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@djblue
#### How should this be tested?
Full build of ui directory
Navigate to new search forms page 
Ensure no regression in search form functionality (sharing, deleting, etc.)

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4170](https://codice.atlassian.net/browse/DDF-4170)
#### Screenshots
<img width="316" alt="screen shot 2018-10-01 at 4 02 31 pm" src="https://user-images.githubusercontent.com/36056781/46321889-cffe8700-c59a-11e8-8936-54600c8276f6.png">
<img width="1919" alt="screen shot 2018-09-28 at 1 50 40 pm" src="https://user-images.githubusercontent.com/36056781/46321891-d42aa480-c59a-11e8-919f-fd28b03ed5ed.png">


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
